### PR TITLE
Ascend ex

### PR DESCRIPTION
**Pull request by kishoian**

### CODE REVIEW

**Review by eric-oconnor — CHANGES_REQUESTED** [review]

**Review by eric-oconnor — COMMENTED** [review]
/qualitycheck

File: content/attacks/posts/2021-12-11-AscendEX.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+title: "AscendEX Hack: $77 Million Stolen in Hot Wallet Breach"
+---
+
+## Summary
+
+On December 11, 2021, AscendEX, a cryptocurrency exchange, was the victim of a hot wallet breach that resulted in the loss of $77 million. The attacker gained access to one of the exchange's hot wallets, which are used to store user funds that are available for withdrawal.
+
+## Attackers
+
+The identity of the attacker or attackers who conducted the hot wallet breach on the AscendEX network remains unknown.
```
Comment (JediFaust) [inline content/attacks/posts/2021-12-11-AscendEX.md:13]: Shorten the sentence

File: content/attacks/posts/2021-12-11-AscendEX.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Wallet addresses that assets were transferred to:
+- ERC20: [0x2c6900b24221de2b4a45c8c89482fff96ffb7e55](https://etherscan.io/address/0x2c6900b24221de2b4a45c8c89482fff96ffb7e55) 
+- Polygon: [0x2c6900b24221de2b4a45c8c89482fff96ffb7e55](https://polygonscan.com/address/0x2c6900b24221de2b4a45c8c89482fff96ffb7e55) 
+- BSC: [0x2c6900b24221de2b4a45c8c89482fff96ffb7e55](https://bscscan.com/address/0x2C6900b24221dE2B4A45c8c89482fFF96FFB7E55) 
+- LTC: [LSvQWLf2kGm7UdXtwKvNj4GU1B4xKWUQXR](https://blockchair.com/litecoin/address/LSvQWLf2kGm7UdXtwKvNj4GU1B4xKWUQXR) 
+- BCH: [qp2x5rnn2fkraxcp4hr6suqmnpdehfaaaqn3tv6jke](https://blockchair.com/bitcoin-cash/address/qp2x5rnn2fkraxcp4hr6suqmnpdehfaaaqn3tv6jke)
+
+## Losses
+
+The attackers managed to steal approximately [$77 million worth](https://twitter.com/peckshield/status/1469915194004766722) of assets from the AscendEX network. This included USDT, BNB, MATIC, and other tokens. The losses were suffered by the AscendEX users who had funds stored in the hot wallet.
```
Comment (JediFaust) [inline content/attacks/posts/2021-12-11-AscendEX.md:24]: Remove last sentence

File: content/attacks/posts/2021-12-11-AscendEX.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Timeline
+
+- **December 11, 2021:** The hot wallet breach on the AscendEX network begins. The attackers gain access to the wallet and withdraw a total of $77 million worth of assets.
```
Comment (JediFaust) [inline content/attacks/posts/2021-12-11-AscendEX.md:28]: Delete first sentence

File: content/attacks/posts/2021-12-11-AscendEX.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- **December 12, 2021:** [The AscendEX team becomes aware of the breach and suspends withdrawals.](https://twitter.com/_AscendEX/status/1470029513551761412)
+- **December 13, 2021:** The AscendEX team reimburses all affected users and announces that they are working with law enforcement to track down the attacker.
```
Comment (JediFaust) [inline content/attacks/posts/2021-12-11-AscendEX.md:30]: Provide source

File: content/attacks/posts/2021-12-11-AscendEX.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+## Security Failure Causes
+
+The AscendEX team has not released any specific details about the security failures that led to the hack.
```
Comment (JediFaust) [inline content/attacks/posts/2021-12-11-AscendEX.md:34]: Make a bullet point, and add one most possible reason

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Review by eric-oconnor — APPROVED** [review]
Quotes added to category

**Comment by JediFaust** [comment]
image

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, merge main into the branch

**Review by JediFaust — APPROVED** [review]

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please resolve conflicts

**Pull request merged**